pushflatpakscript/Dockerfile

```diff
@@ -22,10 +22,14 @@ RUN python -m venv /app \
  && cd /app/configloader \
  && /app/configloader_venv/bin/pip install -r requirements/base.txt \
  && /app/configloader_venv/bin/pip install . \
+ && cd /app \
+ && python -m venv /app/flat_manager_venv \
+ && /app/flat_manager_venv/bin/pip install -r requirements/flat-manager.txt \
+ && curl -O /app/flat_manager_venv/bin/flat-manager-client \
+ https://raw.githubusercontent.com/flatpak/flat-manager/a0005a8046a290590ccb53b631b9949ba52fd8cd/flat-manager-client \
+ && chmod 755 /app/flat_manager_venv/bin/flat-manager-client \
+ && sed -e '1i#!/app/flat_manager_venv/bin/python' -e '1d' /app/flat_manager_venv/bin/flat-manager-client \
```

Comment:
Not sure why this is needed - do you want to enforce the shebang to be the script itself, rather than `python`?

Reply:
Either works. If we don't want to touch the shebang, we should use an array of strings for `flat_manager_client` in `config_schema.json`, and pass that array via json-e in `worker.yml`

Reply:
Never mind me, circling back I understood what you meant here. If we're not overwriting that, it uses a native python which doesn't have none of the virtualenv stuff. I hit that while installing stuff on my local Docker containers yesterday.